libs/sdk-backend-bear/src/convertors/fromBackend/DashboardConverter/scheduledMails.ts

```diff
@@ -59,6 +56,10 @@ export const convertScheduledMail = (
         },
     } = scheduledMail;
 
+    const convertedAttachments = attachments
+        .map(convertScheduledMailAttachment)
+        .filter((a): a is ScheduledMailAttachment => a !== undefined);
```

Comment:
I would use lodash `compact` here instead of `filter` and ideally get rid of the type casting.

Reply:
thanks, it works and handles the types properly